src/main/java/network/radicle/jetbrains/radiclejetbrainsplugin/models/RadDetails.java

```diff
@@ -0,0 +1,65 @@
+package network.radicle.jetbrains.radiclejetbrainsplugin.models;
+
+import java.util.List;
+
+public class RadDetails {
+    private static final String DOUBLE_SPACE = "  ";
+    private static final String SINGLE_SPACE = " ";
+
+    public String id;
+    public String nodeId;
+    public String keyHash;
+    public String keyFull;
+    public String storageGit;
+    public String storageKeys;
+    public String nodeSocket;
+
+    public RadDetails(List<String> details) {
+        parseDetails(details);
+    }
+
+    /* ID             did:key:z6MkgrVBxtgFynk6AWZ9ZDM5NVnHuvWeyMJkVD1aUVtMXPug
```

Comment:
This is DID (Decentralized Identifier), not ID, so can we rename all references to it to match this standard term?